pkg/destroy/libvirt/libvirt_prefix_deprovision.go

```diff
@@ -14,7 +14,7 @@ import (
 // returns true, when the name should be handled.
 type filterFunc func(name string) bool
 
-// ClusterNamePrefixFilter returns true for name
+// ClusterNamePrefixFilter returns true for names
```

Comment:
Same change could be made down on L30.

Reply:
... and it's odd that the AlwaysTrueFilter does not always return true.  :-)

Reply:
> Same change could be made down on L30.

Fixed there too with cd617f8 -> dd82979.